src/index.js

```diff
@@ -27,9 +27,7 @@ let tasks = [
   },
 ];
 
-tasks = _.sortBy(tasks, 'index');
-
-const generateToDoRows = (text, index) => {
+const generateToDoRows = (text, task) => {
```

Comment:
- [ ] OPTIONAL - Consider adding the drag and drop functionality to make the [project interactive](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=The%20user%20should%20be%20able%20to%20reorder%20the%20list%20by%20dragging%20each%20item%20and%20moving%20it%20into%20the%20desired%20position.).

Reply:
I am currently working on it now. Thank you so much, @mikenath223:pray: